week2/homework/src/remove.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+
+function remove(index) {
+  fs.readFile('./todoList.txt', 'utf8', (err, data) => {
+    if (err) throw err;
```

Comment:
It is not wrong to rethrow an error if you think it should be handled by the invoking function, but then you should be consistent. In other files you simply log the error and do not rethrow.

Reply:
i change it , thanks for the info